otx/algorithms/action/adapters/openvino/dataloader.py

```diff
@@ -0,0 +1,230 @@
+"""Data loaders for OpenVINO action recognition models."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+from copy import deepcopy
+from typing import Dict, List
+
+import numpy as np
+from compression.api import DataLoader
+
+from otx.api.entities.annotation import AnnotationSceneEntity
+from otx.api.entities.datasets import DatasetEntity, DatasetItemEntity
+
+
+def get_dataloader(dataset: DatasetEntity, task_type: str, clip_len: int, width: int, height: int) -> DataLoader:
+    """Find proper dataloader for dataset and task type.
+
+    If dataset has only a single video, this returns DataLoader for online demo
+    If dataset has multiple videos, this return DataLoader for academia evaluation
+    """
+    if is_multi_video(dataset):
+        if task_type == "ACTION_CLASSIFICATION":
+            return ActionClsDataLoader(dataset, clip_len, width, height)
+        if task_type == "ACTION_DETECTION":
+            return ActionDetDataLoader(dataset, clip_len, width, height)
+        raise NotImplementedError(f"{task_type} is not supported from action task")
+    return ActionDemoDataLoader
```

Comment:
How about adding `OV` for openvino dataloader? i.e. `ActionClsDataLoader` --> `ActionOVClsDataLoader`.

Reply:
same with `det`, `demo`

Reply:
I think it is a good idea. Thanks